GeneticAlgorithm.py

```diff
@@ -371,6 +385,29 @@ def initialize(self):
 
             self._assignments.append(newCourseAssignment)
 
+            if (len(GAList) - 1) == gata_lab_counter:
```

Comment:
If we go out of bounds for the TAList, initialize back to 0. If not, add 1.

Reply:
I'm requesting changes on this comment because we should be checking if we are out of bounds of TAList and not GAList.